tests/deploys.go

```diff
@@ -0,0 +1,84 @@
+package tests
+
+import (
+	"testing"
+	"time"
+
+	"github.com/nspcc-dev/neo-go/pkg/neotest"
+	"github.com/nspcc-dev/neo-go/pkg/util"
+	"github.com/nspcc-dev/neofs-contract/rpc/nns"
+	"github.com/stretchr/testify/require"
+)
+
+type (
+	// RegisterContractInNNSParams is a param struct for RegisterContractInNNS func.
+	RegisterContractInNNSParams struct {
+		// Name is a contract name. Name must not be a top level domain. For instance: netmap.
```

Comment:
It's not about TLD. It's about collisions with NeoFS contracts. See `rpc/nns/names.go`.